torchtext/experimental/vectors.py

```diff
@@ -233,37 +168,38 @@ def vectors_from_file_object(file_like_object, delimiter=",", unk_tensor=None):
         ValueError: if duplicate tokens are found in FastText file.
 
     """
-    tokens, vectors, dup_tokens = _load_token_and_vectors_from_file(file_like_object.name, delimiter=delimiter)
```

Comment:
Still not a big fan of this "ord(delimiter)" thing. Why not just a string?

Reply:
Reposting my answer from your past comments

> Spoke to the torchscript team and char isn't a supported type. Also thought using a string would be too hacky as there is the potential for the user to pass in multiple characters

Basically, the `std::getline(s, word, static_cast<char>(delimiter_ascii));` expects a char for the delimiter. Do you still think it would be better to use a string in this case?

Reply:
Yes, I think a string would be better and more future proof.